docs/usage.md

```diff
@@ -159,18 +155,18 @@ If `-profile` is not specified, the pipeline will run locally and expect all sof
 - `docker`
   - A generic configuration profile to be used with [Docker](https://docker.com/)
   <!-- - `singularity`
-  - A generic configuration profile to be used with [Singularity](https://sylabs.io/docs/)
-- `podman`
-  - A generic configuration profile to be used with [Podman](https://podman.io/)
-- `shifter`
-  - A generic configuration profile to be used with [Shifter](https://nersc.gitlab.io/development/shifter/how-to-use/)
-- `charliecloud`
-  - A generic configuration profile to be used with [Charliecloud](https://hpc.github.io/charliecloud/)
-- `apptainer`
-  - A generic configuration profile to be used with [Apptainer](https://apptainer.org/)
-- `wave`
-  - A generic configuration profile to enable [Wave](https://seqera.io/wave/) containers. Use together with one of the above (requires Nextflow ` 24.03.0-edge` or later).
-- `conda`
+  - A generic configuration profile to be used with [Singularity](https://sylabs.io/docs/) -->
+  <!-- - `podman`
+  - A generic configuration profile to be used with [Podman](https://podman.io/) -->
+  <!-- - `shifter`
+  - A generic configuration profile to be used with [Shifter](https://nersc.gitlab.io/development/shifter/how-to-use/) -->
+  <!-- - `charliecloud`
+  - A generic configuration profile to be used with [Charliecloud](https://hpc.github.io/charliecloud/) -->
+  <!-- - `apptainer`
+  - A generic configuration profile to be used with [Apptainer](https://apptainer.org/) -->
+  <!-- - `wave`
+  - A generic configuration profile to enable [Wave](https://seqera.io/wave/) containers. Use together with one of the above (requires Nextflow ` 24.03.0-edge` or later). -->
+  <!-- - `conda`
   - A generic configuration profile to be used with [Conda](https://conda.io/docs/). Please only use Conda as a last resort i.e. when it's not possible to run the pipeline with Docker, Singularity, Podman, Shifter, Charliecloud, or Apptainer. -->
```

Comment:
again, this is nf-core standard. the configs will handle this automatically for you

Reply:
As above. Since fastqrepair makes use of a docker env for some of its dependencies, I thought I should include the profiles that I've tested with the package. Can you help me understand?

Reply:
If you use docker the other container engines should just work (except for conda, where we can have a look later to see if you really need docker or if it actually also works with conda